indico/web/forms/fields.py

```diff
@@ -421,7 +421,11 @@ def _value(self):
 class MultipleItemsField(HiddenField):
     """A field with multiple items consisting of multiple string values.
 
-    :param fields: A list of ``(fieldname, title)`` tuples
+    :param fields: A list of ``{'id', 'caption', 'type'}`` dicts. The 'type'
```

Comment:
seems fine. I probably wouldn't use the set-ish literal to describe the keys - you describe them afterwards anyway.